leapp/cli/upgrade/__init__.py

```diff
@@ -175,6 +175,43 @@ def upgrade(args):
         sys.exit(1)
 
 
+@command('preupgrade', help='Generate preupgrade report')
+@command_opt('--whitelist-experimental', action='append', metavar='ActorName',
```

Comment:
just FYI -- is not necessary command_opt is only for -- options - Not that it hurts anyone, it strips them away first anyway and then readds them :D

Reply:
Nice to know. I just copied it from the upgrade command as closely as possible.